tests/monotouch-test/monotouch-test.csproj

```diff
@@ -201,6 +201,7 @@
     <Compile Include="..\common\ConditionalCompilation.cs">
       <Link>ConditionalCompilation.cs</Link>
     </Compile>
+    <Compile Include="CoreFoundation\CFUrlTest.cs" />
```

Comment:
This is not needed, because we have a wildcard include directive further above (in fact it will probably break the build, because csc complains if the same source file is passed twice).

Reply:
yes, it breaks VS4M :(